main.js

```diff
@@ -1,7 +1,3 @@
 const { createServer } = require('./src/createServer');
 
-createServer()
-  .listen(5700, () => {
-    // eslint-disable-next-line no-console
-    console.log('Server started! 🚀');
-  });
+createServer().listen(5700);
```

Comment:
Why u delete this cool information about Server start :////
You can store information about port in variable and first check if it isnt already set in .env file. Like:
const PORT = process.env.PORT || 5700;